src/java.net.http/share/classes/jdk/internal/net/http/ResponseSubscribers.java

```diff
@@ -481,7 +482,12 @@ private ByteBuffer current() throws IOException {
                     if (debug.on()) debug.log("Next Buffer");
                     currentBuffer = currentListItr.next();
                 } catch (InterruptedException ex) {
-                    // continue
+                    try {
+                        close();
+                    } catch (IOException ignored) {
+                    }
```

Comment:
I don't have specific opinion here and this is more of a question - should we be doing:

```
} catch (InterruptedException ex) {
    final IOException toThrow = new IOException(ex);
    try {
        close();
    } catch (IOException ignored) {
        toThrow.addSuppressed(ignored);
    }
    Thread.currentThread().interrupt();
    throw toThrow;    
}
```
i.e. should we be adding any failure to close() as a suppressed exception to the IOException that we throw?

Reply:
I am not sure whether that would be helpful or confusing.

Reply:
I think that's a good point, I don't really have a strong opinion one way or the other on it but would be curious if anyone else does?

Reply:
Let's leave it in the current form then.